lib/src/data/datasource/remote/pokedex_api_service.dart

```diff
@@ -24,6 +25,8 @@ class PokedexApiService {
             (pokemonEntry) async {
           pokemonEntry.pokemon = await _getPokemon(pokemonEntry.entryNumber);
         });
+        pokedexModel.pokemonEntries
+            .removeWhere((pokemonEntry) => pokemonEntry.pokemon == null);
```

Comment:
Why you remove the possible null pokemons?

Reply:
so that the list that I am going to show is only with the non-null pokemons